buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1141,12 +1141,10 @@ artifactId=${project.name}
 
         include "**/*IT.class"
 
-        def pipelineOptionString = configuration.integrationTestPipelineOptions
-        if(configuration.runner?.equalsIgnoreCase('dataflow')) {
+        def pipelineOptionsString = configuration.integrationTestPipelineOptions
+        if(pipelineOptionsString && configuration.runner?.equalsIgnoreCase('dataflow')) {
```

Comment:
Won't you want to create a new pipelineOptionsString if there wasn't one defined containing the args that you added?

Reply:
Not sure if I understand you comment. Are you suggesting we should add our new args, even when pipelineOptionsString starts as empty string? 

I think an empty pipelineOptionsString does not happen in real use case of dataflow task. It only happens when gradle only does evaluation, but nothing else (e.g. ./gradle clean).  In such case I make it this way that pipelineOptionsString stays empty.

Reply:
Your correct in thinking it is unlikely to be empty but I find it odd that we only add some default args if there were args that were specified. Why wouldn't we have the default args all the time?

This is also something that can change later on but its likely that at some point in time someone will be surprised by this behavior.

Reply:
I see. Good question and fair concern.    

Let me think about it and try to run a few other tasks.  IMHO, what we should do depends on if it ONLY happens for 'clean' task.